### PR TITLE
Write some docs on configuration

### PR DESCRIPTION
**Pull request by prophile**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,3 +2,33 @@ SR Comp Mixtape
...
+
+Track configuration
+-------------------
+
+The configuration for a track is a list of triggers, each of which is a dictionary containing the following keys:
+
+- ``start`` is the time of the trigger, relative to the game start time.
+- ``filename`` is the path to a WAV file to play, relative to the mixtape directory.
+- ``output_device`` is the Audio device to send the output to.
+- ``magicq_cue`` is the MagicQ cue ID to send.
```
Comment (RealOrangeOne) [inline README.rst:62]: I think we should define which of these configurations are optional, and which are required _together_ (namely track playback and lighting cues)

File: README.rst [hunk 2]
```diff
@@ -2,3 +2,33 @@ SR Comp Mixtape
...
 
 A service which plays tracks throughout a competition.
+
+Configuration
+-------------
+
+A configuration of tracks and triggers to play during a competition is called
+a ``mixtape``. A mixtape is a directory containing:
+
+- A configuration file, ``playlist.yaml``, which defines trigger points,
```
Comment (RealOrangeOne) [inline README.rst:34]: Meta: Thoughts on adding a dummy `playlist.yaml` file?

File: README.rst [hunk 3]
```diff
@@ -2,3 +2,33 @@ SR Comp Mixtape
...
+Track configuration
+-------------------
+
+The configuration for a track is a list of triggers, each of which is a dictionary containing the following keys:
+
+- ``start`` is the time of the trigger, relative to the game start time.
+- ``filename`` is the path to a WAV file to play, relative to the mixtape directory.
+- ``output_device`` is the Audio device to send the output to.
+- ``magicq_cue`` is the MagicQ cue ID to send.
+- ``magicq_playlist`` is the MagicQ playlist ID to send.
```
Comment (RealOrangeOne) [inline README.rst]: This is actually `magicq_playback`
```suggestion
- ``magicq_playback`` is the MagicQ playlist ID to send.
```

File: README.rst (hunk 2 above)
Comment (WillB97) [reply README.rst:34]: We should

Comment (PeterJCLaw) [reply README.rst:34]: I don't think we should consider this a blocker.

File: README.rst (hunk 1 above)
Comment (RealOrangeOne) [reply README.rst:62]: https://github.com/srobo/srcomp-mixtape/pull/1/commits/2e194d650c7019b7b9763ec910fa00d143c7a597

File: README.rst (hunk 2 above)
Comment (WillB97) [reply README.rst:34]: This should probably be added as a separate PR

**Review by WillB97 — APPROVED** [review]

**Pull request merged**